src/services/sortPeople.ts

```diff
@@ -0,0 +1,69 @@
+import { Person } from '../types';
+
+export const sortingPeople = (
+  sortType: string | null,
+  orderType: string | null,
+  filteredPeople: Person[] | [],
+) => {
+  let pervPeople = filteredPeople;
+
+  if (!sortType) {
+    pervPeople = filteredPeople;
```

Comment:
The variable `pervPeople` is initialized with `filteredPeople` and reassigned the same value if `!sortType`. This is redundant and can be simplified by returning `filteredPeople` directly if `!sortType`.